<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillWindow</class>
 <widget class="QMainWindow" name="BillWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="billtableView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>511</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="accountListView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>21</y>
      <width>71</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveBillButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>创建账单</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>如果账单已完成，请及时删除！</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupersonal_financial_manager">
    <property name="title">
     <string>personal financial manager 账单提醒</string>
    </property>
   </widget>
   <addaction name="menupersonal_financial_manager"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
